docs/reference/9-analytics-middle-tier/contributor-guide/contributions/unit-test-framework.md

```diff
@@ -0,0 +1,611 @@
+# Unit Test Framework
+
+## Introduction
+
+The Analytics Middle Tier source code contains extensive unit testing of the
+views, utilizing a custom framework for loading and refreshing test data on all
+supported Data Standards against a SQL Server database and a Postgres database.
+These unit tests serve the dual purposes of exploring the required data during
+development and providing regression tests for safely refactoring views and
+correcting bugs.
+
+All views created by the core Ed-Fi Analytics development team have a complete
+set of unit tests that cover:
+
+* Output columns
+* Where clause conditions
+* Join clause conditions
+
+The development team encourages [community
+contributors](../readme.md) to develop similar unit tests for
+their views. The following sections provide a walkthrough of how to add unit
+tests for a simple use case, as well as detailed information about the framework
+architecture for those who would like to know more.
+
+With the recent addition of having test cases execute against both SQL Server
+and Postgres, after downloading the Analytics Middle Tier you should review the
+[Analytics Middle Tier Test project
+README](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Analytics-Middle-Tier/blob/main/src/EdFi.AnalyticsMiddleTier.Tests/readme.md)
+and  [this page in Tech
+Docs](https://edfi.atlassian.net/wiki/spaces/EFTD/pages/24807117) that mentions
+the additional setup and caveats to testing against Postgres in addition to SQL
+Server.
```

Comment:
This links to a Tech Docs page under Analytics Middle Tied Enhancements: https://edfi.atlassian.net/wiki/x/MIJ6AQ 

which is different from the AMT pages being moved currently: https://edfi.atlassian.net/wiki/x/oghwAQ

Does this portion also need to be moved over?

Reply:
At this time I'm not planning on moving the Technical Designs ("EFTD") space out of Confluence, so this link does not need to change...

Oh, but the content itself is outdated. Let's make a note of this for continued improvement without needing to hold up this PR.